app/org/sagebionetworks/bridge/validators/SurveyValidator.java

```diff
@@ -205,6 +208,41 @@ private void doValidateConstraintsType(Errors errors, UIHint hint, MultiValueCon
             rejectField(errors, "uiHint",
                     "doesn't allow multiples but the '%s' UI hint gathers more than one answer", hintName);
         }
+
+        // must have an enumeration (list of question options)
+        List<SurveyQuestionOption> optionList = mcon.getEnumeration();
+        if (optionList == null || optionList.isEmpty()) {
+            rejectField(errors, "enumeration", "must have non-null, non-empty choices list");
```

Comment:
As per IM conversation, this is necessary because otherwise, we get duplicate column names in Synapse.

While we could implement dedupe logic, this adds extra complexity, is very unlikely to be relevant, and researchers and trivially work around this.